### ISSUE TITLE
Rendering problem on 0.37.3 (Linux)

### ISSUE THREAD

**Issue by geocool** [atom/electron#4984]
* Electron version: 0.37.3
* Operating system: elementary OS Freya (based on ubuntu 14.04 LTS)

**The current version of electron has problem rendering the window.**
This is a screenshot after re-sizing dev tools a little bit and marking some words.
![screenshot from 2016-04-01 13 03 02](https://cloud.githubusercontent.com/assets/2379934/14204156/f27be81e-f80a-11e5-917a-0810824cc6fe.png)
  
 
This is a screenshot of the same app using electron 0.36.12
![screenshot from 2016-04-01 13 02 21](https://cloud.githubusercontent.com/assets/2379934/14204160/f8b46c9c-f80a-11e5-9c78-4d48661df340.png)

**Comment by ficristo** [comment]
I think I have the same problem of blurry font on Windows 7.

**Comment by deepak1556** [comment]
This is related to https://github.com/atom/electron/issues/4950 , the problem should go away if you provide a background color.

**Comment by ficristo** [comment]
I can confirm setting the background color on the html side fixed it on Windows.
I tried using the BrowserWindow backgroundColor option but it did nothing.
@deepak1556 Thank you.

**Comment by danielroehrig** [comment]
I have a similar issue (also using elementary OS) with Atom, GitKraken und Steam with really blurry fonts. I posted [this question](https://elementaryos.stackexchange.com/questions/4951/blurry-fonts-in-atom-steam-and-gitkraken) on the elementary OS Stack Exchange, but was told that this is a bug in electron. Since I can't change the background in Atom or Steam, I am stuck with a really ugly edito,r I guess.

**Comment by denysvitali** [comment]
Same here, using [this boilerplate](https://github.com/szwacz/electron-boilerplate) I got [the mentioned problem](http://i.imgur.com/8JeUhPU.png).

Fixed by adding a `background-color: #FFF;` to the body element

**Comment by paulcbetts** [comment]
@danielroehrig I think your bug is a different bug, that looks like the device scale that you're running at is != the device scale that Chrome thinks it is. What's `window.devicePixelRatio` compared to your DPI scale factor?

**Comment by danielroehrig** [comment]
`window.devicePixelRatio` return 1. I don't know how to determine the DPI scale factor.

**Comment by paulcbetts** [comment]
`xdpyinfo | grep dots`, take that number and divide by 96

**Comment by danielroehrig** [comment]
`resolution:    96x96 dots per inch` divided by 96 = 1, and opening the developer tools in atom and asking for `window.devicePixelRatio` also return 1.

**Comment by paulcbetts** [comment]
Hm odd, and `gsettings get org.gnome.desktop.interface scaling-factor` isn't set?

**Comment by danielroehrig** [comment]
```
gsettings get org.gnome.desktop.interface scaling-factor
uint32 0
```
So, no.

**Comment by paulcbetts** [comment]
What's the native resolution of your monitor and its size? You might fix this by setting `gsettings set org.gnome.desktop.interface scaling-factor 2`

**Comment by danielroehrig** [comment]
Nope, that only makes everything twice as big. I have a 1920x1080 monitor, so no high-dpi here that could be responsible for the blurry fonts.

**Comment by zcbenz** [comment]
Closed with #4993.
